inventory-service/src/main/java/com/petclinic/inventoryservice/presentationlayer/InventoryController.java

```diff
@@ -128,5 +128,10 @@ public Mono<ResponseEntity<InventoryTypeResponseDTO>> addInventoryType(@RequestB
                 .defaultIfEmpty(ResponseEntity.notFound().build());
     }
 
+    @GetMapping("/type")
+    public Flux<InventoryTypeResponseDTO> getAllInventoryTypes(){
```

Comment:
Shouldn't you be returning a ResponseEntity, with the 200 OK?

Reply:
When you return a Flux, the framework should take care of serializing the data into the appropriate format and setting the appropriate HTTP response headers, like the Content-Type. We should not need to create a ResponseEntity manually while using Flux unless I am clueless.